clippy_utils/src/msrvs.rs

```diff
@@ -69,19 +70,20 @@ msrv_aliases! {
     1,35,0 { OPTION_COPIED, RANGE_CONTAINS }
     1,34,0 { TRY_FROM }
     1,33,0 { UNDERSCORE_IMPORTS }
-    1,32,0 { CONST_IS_POWER_OF_TWO }
+    1,32,0 { CONST_IS_POWER_OF_TWO, DURATION_CONST_FROM_NANOS_MICROS_MILLIS_SECS }
```

Comment:
nit: the other constants have `CONST_` at the start
```suggestion
    1,32,0 { CONST_IS_POWER_OF_TWO, CONST_DURATION_FROM_NANOS_MICROS_MILLIS_SECS }
```

Reply:
upsi, fixed